admin/includes/classes/AdminRequestSanitizer.php

```diff
@@ -523,7 +553,11 @@ private function filterMultiDimensional($parameterName, $parameterDefinition)
         if (!isset($requestPost[$parameterName])) {
             return;
         }
+        // Add the parameterName to the base arrayname.
+        $this->arrayName = $this->setCurrentArrayName($parameterName, $this->arrayName);
+        $currentArrayName = $this->arrayName;
```

Comment:
This and the above line can be combined into a single statement with the far left being `$currentArrayName` which will be assigned the value to `$this->arrayName` which will be assigned the result of `$this->setCurrentArrayName`. Perhaps though so many variables aren't necessary?